src/nfs.c

```diff
@@ -495,18 +514,18 @@ static void nfs_read_linux(FILE *fh, const char *inst) {
     if (fields_num < 3)
       continue;
 
-    if (strcmp(fields[0], "proc2") == 0) {
+    if (strcmp(fields[0], "proc2") == 0 && ignore_v2 == 0) {
```

Comment:
With boolean variables (`_Bool`), please use `!ignore_v2` instead. Here and below.